testOPHD/testOPHD.cpp

```diff
@@ -0,0 +1,6 @@
+#include <gtest/gtest.h>
+
+
+TEST(Test, EmptyTest)
+{
+}
```

Comment:
Slightly awkward situation, as we still haven't completed:
- Issue #1422

Ideally, `MapOffset` would be migrated to `libOphd`, and so unit test code for it would go into `testLibOphd`. Unfortunately that hasn't happened yet.

----

In C++ it's a bit more natural to write unit tests against library code. I wouldn't expect there to be a `testOphd` project, since the `Ophd` project is an app rather than a library.

----

The `makefile` doesn't support building the new project. It does support building `testLibOphd`.